[2025-04-21T13:43:28.819+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-21T13:43:28.853+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hourly_etl_pipeline.load_stg scheduled__2025-04-21T12:00:00+00:00 [queued]>
[2025-04-21T13:43:28.866+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hourly_etl_pipeline.load_stg scheduled__2025-04-21T12:00:00+00:00 [queued]>
[2025-04-21T13:43:28.866+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-04-21T13:43:28.887+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_stg> on 2025-04-21 12:00:00+00:00
[2025-04-21T13:43:28.893+0000] {standard_task_runner.py:63} INFO - Started process 784 to run task
[2025-04-21T13:43:28.898+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hourly_etl_pipeline', 'load_stg', 'scheduled__2025-04-21T12:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/hourly_etl_pipeline.py', '--cfg-path', '/tmp/tmp1ddnr25k']
[2025-04-21T13:43:28.902+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask load_stg
[2025-04-21T13:43:29.038+0000] {task_command.py:426} INFO - Running <TaskInstance: hourly_etl_pipeline.load_stg scheduled__2025-04-21T12:00:00+00:00 [running]> on host ab2b55c0754d
[2025-04-21T13:43:29.192+0000] {logging_mixin.py:188} WARNING - /root/.local/lib/python3.8/site-packages/***/utils/context.py:212 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-04-21T13:43:29.277+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hourly_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_stg' AIRFLOW_CTX_EXECUTION_DATE='2025-04-21T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-21T12:00:00+00:00'
[2025-04-21T13:43:29.279+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-21T13:43:29.303+0000] {logging_mixin.py:188} INFO - Target time: 2025-04-21 12:00:00+00:00
[2025-04-21T13:43:29.336+0000] {base.py:84} INFO - Using connection ID 'postgresql_lab08' for task execution.
[2025-04-21T13:43:29.372+0000] {logging_mixin.py:188} INFO - Processing file: year=2025/month=04/day=21/hour=12/browser_events.jsonl.zip
[2025-04-21T13:43:29.391+0000] {base.py:84} INFO - Using connection ID 's3_yandex' for task execution.
[2025-04-21T13:43:29.393+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='s3_yandex', conn_type='aws') credentials retrieved from login and password.
[2025-04-21T13:44:22.914+0000] {logging_mixin.py:188} INFO - Processed successfully: year=2025/month=04/day=21/hour=12/browser_events.jsonl.zip
[2025-04-21T13:44:22.915+0000] {logging_mixin.py:188} INFO - Processing file: year=2025/month=04/day=21/hour=12/device_events.jsonl.zip
[2025-04-21T13:45:19.056+0000] {logging_mixin.py:188} INFO - Processed successfully: year=2025/month=04/day=21/hour=12/device_events.jsonl.zip
[2025-04-21T13:45:19.057+0000] {logging_mixin.py:188} INFO - Processing file: year=2025/month=04/day=21/hour=12/geo_events.jsonl.zip
[2025-04-21T13:46:14.023+0000] {logging_mixin.py:188} INFO - Processed successfully: year=2025/month=04/day=21/hour=12/geo_events.jsonl.zip
[2025-04-21T13:46:14.024+0000] {logging_mixin.py:188} INFO - Processing file: year=2025/month=04/day=21/hour=12/location_events.jsonl.zip
[2025-04-21T13:47:06.872+0000] {logging_mixin.py:188} INFO - Processed successfully: year=2025/month=04/day=21/hour=12/location_events.jsonl.zip
[2025-04-21T13:47:06.876+0000] {logging_mixin.py:188} INFO - Loaded successfully
[2025-04-21T13:47:06.879+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-21T13:47:06.879+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-21T13:47:06.913+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hourly_etl_pipeline, task_id=load_stg, run_id=scheduled__2025-04-21T12:00:00+00:00, execution_date=20250421T120000, start_date=20250421T134328, end_date=20250421T134706
[2025-04-21T13:47:06.948+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-21T13:47:06.983+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-21T13:47:06.988+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
